notebooks/curation_api/README.md

```diff
@@ -0,0 +1,24 @@
+# Discover API notebooks
+
+These notebooks facilitate use of the [Discover API](https://api.cellxgene.cziscience.com/curation/ui/#/) which enables 
+data consumers to query for public collections and datasets available in 
+[CZ CELLxGENE Discover](https://cellxgene.cziscience.com/) and to download individual datasets.
+
+## Notebook usage
+
+Many notebooks have a few fields that require user input, such as the relevant Collection id or Dataset id.
+
+### Python notebooks: `python/`
+
+These notebooks are the most user-friendly; start here if you're unsure. All access token generation, url construction, and response 
```

Comment:
_access token generation_ is confusing for non-curation use which does not require access tokens?